index.html

```diff
@@ -10,23 +10,215 @@
+      </div>
+      <div class="languages">
+        <ul class="languages-list">
+          <li class="lang">html</li>
+          <li class="lang">css</li>
+          <li class="lang">javaScript</li>
+        </ul>
+      </div>
+      <div class="project-access">See project</div>
+    </div>
+
+    <div class="project" id="project2">
+      <div class="pimage"></div>
+      <div class="p-title">
+        <h2>Multi-Post Stories</h2>
+      </div>
+      <div class="projectdetails">
+        <ul class="details-list">
+          <li class="client">CANOPY</li>
+          <li class="spacedot"></li>
+          <li class="role">Back End Dev</li>
+          <li class="spacedot"></li>
+          <li class="year">2015</li>
+        </ul>
+      </div>
+      <div class="p-description">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <div class="languages">
+        <ul class="languages-list">
+          <li class="lang">html</li>
+          <li class="lang">css</li>
+          <li class="lang">javaScript</li>
+        </ul>
+      </div>
+      <div class="project-access">See project</div>
+    </div>
+
+     <div class="project" id="project3">
+      <div class="pimage"></div>
+      <div class="p-title">
+        <h2>Tonic</h2>
+      </div>
+      <div class="projectdetails">
+        <ul class="details-list">
+          <li class="client">CANOPY</li>
+          <li class="spacedot"></li>
+          <li class="role">Back End Dev</li>
+          <li class="spacedot"></li>
+          <li class="year">2015</li>
+        </ul>
+      </div>
+      <div class="p-description">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <div class="languages">
+        <ul class="languages-list">
+          <li class="lang">html</li>
+          <li class="lang">css</li>
+          <li class="lang">javaScript</li>
+        </ul>
+      </div>
+      <div class="project-access">See project</div>
+    </div>
+
+    <div class="project" id="project4">
+      <div class="pimage"></div>
+      <div class="p-title">
+        <h2>Multi-Post Stories</h2>
+      </div>
+      <div class="projectdetails">
+        <ul class="details-list">
+          <li class="client">CANOPY</li>
+          <li class="spacedot"></li>
+          <li class="role">Back End Dev</li>
+          <li class="spacedot"></li>
+          <li class="year">2015</li>
+        </ul>
+      </div>
+      <div class="p-description">
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      </div>
+      <div class="languages">
+        <ul class="languages-list">
+          <li class="lang">html</li>
+          <li class="lang">css</li>
+          <li class="lang">javaScript</li>
+        </ul>
+      </div>
+      <div class="project-access">See project</div>
```

Comment:
- [ ] The same thing here, please check the previous comment ⬆

Reply:
Roger that!